### ISSUE TITLE
Hamburger menu on mobile does not change to "close" icon when the navigation is opened

### ISSUE THREAD

**Issue by Tajana-Somogyi** [ALASdoo/alasdoo_com_v2#154]
- [x] "close" icon replaces hamburger menu
- [x] alt text should be "open menu" and "close menu"

**Comment by TheKolega** [comment]
Looks good :green_circle:

**Comment by Tajana-Somogyi** [comment]
Focus should loop inside the nav menu (after the last nav link it should return to the close button). Otherwise, the navigation menu covers the focus on the rest of the page.
